cmd/config-manager/main.go

```diff
@@ -180,22 +180,30 @@ func detectRuntime() (string, *dbus.Conn, error) {
 	// (either containerd or CRI-O).If more than one container runtime systemd unit is found
 	// to be in an active state, the process fails.
 	log.Infof("looking for active runtime units on D-Bus...")
-	units, err := conn.ListUnitsByPatternsContext(context.Background(), []string{"active"}, []string{containerdUnit, crioUnit})
+	units, err := conn.ListUnitsFilteredContext(context.Background(), []string{"active"})
+
+	expectedUnits := make([]dbus.UnitStatus, 0)
+	for _, unit := range units {
+		if unit.Name == "containerd.service" || unit.Name == "crio.service" {
```

Comment:
nitpick: There are `containerdUnit` and `crioUnit` consts defined for these, so probably those should be used here instead.

Reply:
Good catch. Fixed.